manifests/deploy/deployment.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: combo
+  name: combo-operator
+  labels:
+    app: combo
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: combo-operator
+  template:
+    metadata:
+      labels:
+        app: combo-operator
+    spec:
+      serviceAccountName: combo-operator
+      containers:
+      - name: combo
+        image: quay.io/operator-framework/combo:latest
+        imagePullPolicy: Never
```

Comment:
By default, the imagePullPolicy is set to 'Always' implicitly anytime the `latest` tag is present. In any case, we should avoid using the 'Never' policy here and stick with the implicit 'Always' or specify 'IfNotPresent'.

Reply:
Set a `IfNotPresent` here so that you can install locally and use a kind-loaded image without needing to pull.